docs/guides/gutenberg.md

```diff
@@ -43,7 +43,7 @@ function my_acf_init() {
 Next, you you have to create your `render_callback()` function:
 
 ```php
-function my_acf_block_render_callback( $block ) {
+function my_acf_block_render_callback( $block, $content = '', $is_preview ) {
```

Comment:
Since $content has a default value, $is_preview should default to true or false to prevent PHP throwing an error

Reply:
oh! and the docblocks you left in #1885 also can go here